download.go

```diff
@@ -357,7 +359,7 @@ func downloadFile(f *gdrive.File, localPath string, progressBar *pb.ProgressBar)
 
 	verbose.Printf("Downloaded and wrote %d bytes to %s", f.FileSize, localPath)
 
-	return os.Chtimes(localPath, normalizeModTime(f.ModTime), normalizeModTime(f.ModTime))
+	return err
```

Comment:
Won't this return the 'err' from the "writeCloser, err := ..." line at the top of the function?  I think here you just want "return"?  (But even then, it's not completely to clear if the value of err returned from the function is in fact modify-able by the defer statement.

Instead, how about removing the "defer writeCloser.Close()" from the original function, manually closing it in the case where downloadDriveFile returns an error, and also closing it before the Chtimes call?